### PR TITLE
Added 12-hour time

### PR DESCRIPTION
**Pull request by PhilrocWP**
Title says it all.

### CODE REVIEW

**Comment by fincs** [comment]
Uh, what you wrote is not valid C source code.

**Comment by mtheall** [comment]
C identifiers cannot start with a decimal digit.

File: source/statusbar.c [hunk 1]
```diff
@@ -30,12 +30,39 @@ void drawStatusBar(bool wifiStatus, bool charging, int batteryLevel)
...
+	if(24hour === 20)
+		12hour === 8
+	if(24hour === 21)
+		12hour === 9
+	if(24hour === 22)
+		12hour === 10
+	if(24hour === 23)
+		12hour === 11
+	if(24hour === 24)
+		12hour === 12
```
Comment (mtheall) [inline source/statusbar.c]: Can't this just be:
```
u8 _12hour = (_24hour % 12) + 12;

Comment (mtheall) [inline source/statusbar.c]: Can't this just be:
```
u8 _12hour = _24hour % 12;
if(_12hour == 0)
    _12hour = 12;
```

**Comment by PhilrocWP** [comment]
mtheall: But then you would be getting numbers that weren't whole.

**Comment by HandsomeMatt** [comment]
just round them down, oh my.

**Comment by PhilrocWP** [comment]
I'm kind of a noob at C...

**Comment by HandsomeMatt** [comment]
read up, take the suggestions in and recommit

**Comment by PhilrocWP** [comment]
i have recommitted, see what I did.

**Comment by HandsomeMatt** [comment]
Use the modulus suggested

**Comment by PhilrocWP** [comment]
OOOOOHHHHHHHHH.

**Comment by PhilrocWP** [comment]
I have recommitted (again), see what I did.

**Comment by smealum** [comment]
i appreciate the effort but i really don't think we need to display the time twice, sorry.

**Pull request closed without merging**